packages/transformers/js/src/JSTransformer.js

```diff
@@ -101,6 +101,9 @@ const CONFIG_SCHEMA: SchemaEntity = {
         },
       ],
     },
+    unstable_inlineConstants: {
```

Comment:
Not sure we need the `unstable` prefix here. Should I remove?

Reply:
Depends on how confident we are in the implementation, or do we want to test it in Jira first?

Reply:
I've already tested Jira locally. I think let's keep it as unstable for now as it gives us the most flexibility.